process/smartContract/process.go

```diff
@@ -600,37 +603,29 @@ func (sc *scProcessor) processSCRForSender(
 	txHash []byte,
 	vmInput *vmcommon.ContractCallInput,
 	vmOutput *vmcommon.VMOutput,
-	acntSnd state.UserAccountHandler,
 	lockedGas uint64,
-) (*smartContractResult.SmartContractResult, error) {
+) (*smartContractResult.SmartContractResult, *smartContractResult.SmartContractResult, error) {
```

Comment:
Although this works, returning 2 consecutive elements of the same type is always prone to faults (caused by switching variables)